### PR TITLE
Implement a method to get all the Bookies

### PR DESCRIPTION
**Pull request by mino181295**
In this PR I'm adding a `BokKeeperAdmin#getAllBookies` to retrive all the bookies (cookie) retriving them from the `RegistrationClient`

This is still a work in progress, I'm adding some tests in the next commit.

### CODE REVIEW

**Comment by eolivelli** [comment]
Please add tests and check CI failures

**Comment by jvrao** [comment]
Looks fine to me and it really doesn't intercept what we are doing. I guess tests are needed

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java [hunk 1]
```diff
@@ -395,6 +395,7 @@ public void testListBookiesCmdReadOnly() throws Exception {
             .apply(same(shell.bkConf), same(mockListBookiesFlags));
         verify(mockListBookiesFlags, times(1)).readonly(true);
         verify(mockListBookiesFlags, times(1)).readwrite(false);
+        verify(mockListBookiesFlags, times(1)).all(true);
```
Comment (eolivelli) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java]: Is this test passing? I would expect 'false' here

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java [hunk 2]
```diff
@@ -408,5 +409,21 @@ public void testListBookiesCmdReadWrite() throws Exception {
             .apply(same(shell.bkConf), same(mockListBookiesFlags));
         verify(mockListBookiesFlags, times(1)).readonly(false);
         verify(mockListBookiesFlags, times(1)).readwrite(true);
+        verify(mockListBookiesFlags, times(1)).all(false);
```
Comment (eolivelli) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java]: Is this test passing? I would expect 'false' here

Comment (mino181295) [reply bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieShellTest.java]: I think that if you pass "listbookies", "-rw"  the result is the available read/write bookies
If you pass "listbookies", "-a" you see all the bookies (including the readonly/unavailable)

**Comment by eolivelli** [comment]
@mino181295 
I am seeing this error on CI on a precommit job, can you please check and send a fix ?

```
2019-07-16 12:06:26,599 - ERROR - [main:DiscoveryCommand@61] - Fail to process command 'list'
java.lang.NullPointerException
	at org.apache.bookkeeper.common.concurrent.FutureUtils.result(FutureUtils.java:72)
	at org.apache.bookkeeper.common.concurrent.FutureUtils.result(FutureUtils.java:61)
	at org.apache.bookkeeper.tools.cli.commands.bookies.ListBookiesCommand.run(ListBookiesCommand.java:103)
	at org.apache.bookkeeper.tools.cli.commands.bookies.ListBookiesCommand.run(ListBookiesCommand.java:39)
	at org.apache.bookkeeper.tools.cli.helpers.DiscoveryCommand.apply(DiscoveryCommand.java:57)
	at org.apache.bookkeeper.tools.cli.helpers.ClientCommand.apply(ClientCommand.java:54)
	at org.apache.bookkeeper.tools.common.BKCommand.apply(BKCommand.java:94)
	at org.apache.bookkeeper.tools.common.BKCommand.lambda$apply$0(BKCommand.java:49)
	at org.apache.bookkeeper.tools.framework.Cli.run(Cli.java:224)
	at org.apache.bookkeeper.tools.framework.Cli.runCli(Cli.java:244)
	at org.apache.bookkeeper.tools.common.BKCommand.apply(BKCommand.java:51)
	at org.apache.bookkeeper.tools.cli.commands.bookies.ListBookiesCommandTest.testCommand(ListBookiesCommandTest.java:157)
	at org.apache.bookkeeper.tools.cli.commands.bookies.ListBookiesCommandTest.testListNoArgs(ListBookiesCommandTest.java:135)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
```

**Pull request merged**